### PR TITLE
Refactor CSS

### PR DESCRIPTION
**Pull request by dfosco**
- [x] Refactor file structure to use `ComponentName/index.js` / `ComponentName/style.scss`
 - [x] Use BEM syntax (to avoid conflicts in the future)
 - [x] Install `vtex-css-utils`
 - [x] Organize utils folder and usage

### CODE REVIEW

**Comment by dfosco** [comment]
@brenoc @tamorim @bellacorbo Are we good to close this?

**Comment by brenoc** [comment]
@dfosco can you rebased it please?

**Comment by dfosco** [comment]
@brenoc Yes :relieved:

**Comment by brenoc** [comment]
@dfosco you should add to the body of your commit

> Fix #17

**Comment by dfosco** [comment]
@brenoc Rebased! No conflicts found :D

**Comment by dfosco** [comment]
@brenoc @tamorim Needs review

File: src/components/Banner/BannerImage/style.scss [hunk 1]
```diff
@@ -0,0 +1,5 @@
+@import "~utils/base/base";
```
Comment (brenoc) [inline src/components/Banner/BannerImage/style.scss]: You can remove this for faster build time.

File: src/components/Banner/style.scss [hunk 2]
```diff
@@ -0,0 +1,5 @@
+@import "~utils/base/base";
```
Comment (brenoc) [inline src/components/Banner/style.scss]: You can remove this for faster build time.

**Comment by dfosco** [comment]
@brenoc Done

**Comment by brenoc** [comment]
:+1:

**Comment by brenoc** [comment]
Dammit there's a merge there :(

**Comment by brenoc** [comment]
@dfosco the branch feature/refactor-css can be deleted now?

**Comment by dfosco** [comment]
@brenoc Yes, we're done here!

**Pull request merged**